python/pyarrow/hdfs.py

```diff
@@ -168,6 +168,8 @@ def connect(host="default", port=0, user=None, kerb_ticket=None,
     driver : {'libhdfs', 'libhdfs3'}, default 'libhdfs'
       Connect using libhdfs (JNI-based) or libhdfs3 (3rd-party C++
       library from Apache HAWQ (incubating) )
+    extra_conf : dict of extra Key/Value pairs for config; Will override
```

Comment:
This should be "dict, default None", and then the rest of the documentation go on the following line

Reply:
Fixed with the most recent commit :)